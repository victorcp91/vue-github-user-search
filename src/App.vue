<template>
  <div id="app">
    <router-view></router-view>
  </div>
</template>

<script>
import start from "./components/main/Start";

export default {
  name: 'app'
}
</script>

<style lang="scss">
#app {
  max-width: 1200px;
  width: 100%;
  margin:60px auto 0 auto;
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  text-align: center;
  color: black;    
}
</style>
