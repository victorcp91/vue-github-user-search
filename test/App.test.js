import { shallow, createLocalVue } from 'vue-test-utils';
import App from '../src/App';
import VueRouter from 'vue-router';

describe('App.test.js', () => {
    const localVue = createLocalVue();
    localVue.use(VueRouter);

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(App, {
            localVue
        })
    });

    it('should match html structure', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});