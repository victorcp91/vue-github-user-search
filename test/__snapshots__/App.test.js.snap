// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.test.js should match html structure 1`] = `
<div
  id="app"
>
  <!---->
</div>
`;
